docs/framework/migration-guide/application-compatibility.md

```diff
@@ -1,142 +1,110 @@
-  
--   [Application Compatibility in 4.6.2](../../../docs/framework/migration-guide/application-compatibility-in-the-net-framework-4-6-2.md)  
-
-- [Application Compatibility in 4.7](../../../docs/framework/migration-guide/application-compatibility-in-the-net-framework-4-7.md)
-  
-## Migrating from the .NET Framework 4.6.1  
- If you are migrating from the .NET Framework 4.6.1, review the following topics for application compatibility issues that may affect your app:
- 
-- [Application Compatibility in 4.6.2](../../../docs/framework/migration-guide/application-compatibility-in-the-net-framework-4-6-2.md)
-
-- [Application Compatibility in 4.7](../../../docs/framework/migration-guide/application-compatibility-in-the-net-framework-4-7.md)  
-
-## Migrating from the .NET Framework 4.6.2
-
-If you are migrating from the .NET Framework 4.6.2, review [Application Compatibility in 4.7](../../../docs/framework/migration-guide/application-compatibility-in-the-net-framework-4-7.md) for application compatibility issues that may affect your app.  
-
-## See Also  
- [Versions and Dependencies](../../../docs/framework/migration-guide/versions-and-dependencies.md)   
- [What's New](../../../docs/framework/whats-new/index.md)   
- [What's Obsolete in the Class Library](../../../docs/framework/whats-new/whats-obsolete.md)
+
+## Introduction
+Compatibility is a very important goal of each .NET release. Compatibility
+ensures that each version is additive, so previous versions will still work. On
+the other hand, changes to previous functionality (to improve performance,
+address security issues, or fix bugs) can cause compatibility problems in
+existing code or existing applications that run under a later version. The .NET
+Framework recognizes retargeting changes and runtime changes. Retargeting
+changes affect applications that target a specific version of the .NET Framework
+but are running on a later version. Runtime changes affect all applications
+running on a particular version.
+
+Each app targets a specific version of the .NET Framework, which can be specified by:
+
+* Defining a target framework in Visual Studio.
+* Specifying the target framework in a project file.
+* Applying a <xref:System.Runtime.Versioning.TargetFrameworkAttribute> to the source code.
+
+When running on a newer version than what was targeted, the .NET Framework will
+use quirked behavior to mimic the older targeted version. In other words, the
+app will run on the newer version of the Framework, but act as if it's running
+on the older version. Many of the compatibility issues between versions of the .NET
+Framework are mitigated through this quirking model.
+
+## Runtime changes
+
+Runtime issues are those that arise when a new runtime is placed on a machine
+and the same binaries are run, but different behavior is seen. If a binary was
+compiled for .NET 4.0 it will run in .NET 4.0 compatibility mode on 4.5+. Many
```

Comment:
.NET -> .NET Framework
4.5+ -> couldn't find a definitive answer on the style guide about this but I don't think 4.5+ localizes well. My recommendation would be to change this to 4.5 or later versions, for example.